examples/book-with-adult-and-infant.py

```diff
@@ -0,0 +1,111 @@
+from datetime import date
+
+from duffel_api import Duffel
+
+
+if __name__ == "__main__":
+    print("Duffel Flights API - book with adult and infant example")
+    client = Duffel()
+    departure_date = date.today().replace(date.today().year + 1)
+    slices = [
+        {
+            "origin": "LHR",
+            "destination": "STN",
+            "departure_date": departure_date.strftime("%Y-%m-%d"),
+        },
+    ]
+    offer_request = (
+        client.offer_requests.create()
+        .passengers(
+            [{"type": "adult"}, {"age": 1}, {"age": (date.today().year - 2003)}]
+        )
+        .slices(slices)
+        .execute()
+    )
+
+    print("Created offer request: %s" % (offer_request.id))
+
+    offers = client.offers.list(offer_request.id)
+    offers_list = list(enumerate(offers))
+
+    print("Got %d offers" % len(offers_list))
+
+    selected_offer = offers_list[0][1]
+
+    print("Selected offer %s to book" % (selected_offer.id))
+
+    priced_offer = client.offers.get(selected_offer.id)
+
+    print(
+        "The final price for offer %s is %s (%s)"
+        % (priced_offer.id, priced_offer.total_amount, priced_offer.total_currency)
+    )
+
+    payments = [
+        {
+            "currency": selected_offer.total_currency,
```

Comment:
```suggestion
            "currency": priced_offer.total_currency,
```

I would use the same offer variable so there's no question as to "why not use the same variable?".